docker/ray/Dockerfile

```diff
@@ -10,6 +10,9 @@ COPY requirements_compiled.txt ./
 COPY $WHEEL_PATH .
 COPY $FIND_LINKS_PATH $FIND_LINKS_PATH
 
+# Required a recent version of setuptools to be compatible with python 3.12+.
+RUN $HOME/anaconda3/bin/pip install setuptools==71.1.0
```

Comment:
maybe install this right after conda is installed? so that it can be cached?

we would like to keep this file to be only installing ray.

Reply:
totally